cfbs/validate.py

```diff
@@ -166,6 +170,46 @@ def _validate_top_level_keys(config):
             )
 
 
+def validate_module_name_content(name):
+    MAX_MODULE_NAME_LENGTH = 64
+
+    if len(name) > MAX_MODULE_NAME_LENGTH:
+        raise CFBSValidationError(
+            name,
+            "Module name is too long (over "
+            + str(MAX_MODULE_NAME_LENGTH)
+            + " characters)",
+        )
+
+    # lowercase ASCII alphanumericals, starting with a letter, and possible singular dashes in the middle
+    r = "[a-z][a-z0-9]*(-[a-z0-9]+)*"
+    proper_name = name
+
+    if is_module_local(name):
+        if not name.startswith("./"):
+            raise CFBSValidationError(name, "Local module names should begin with `./`")
```

Comment:
This is ensure by `is_module_local()`, so should be an assert / programmer error if that's not the case:

 ```suggestion
        assert name.startswith("./"), "is_module_local() should ensure this module starts with ./"
```